_conferences/droidcon-sf-2023.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Droidcon"
+website: https://sf.droidcon.com
+location: San Francisco, CA, USA
+
+date_start: 2023-06-08
+date_end:   2023-06-09
```

Comment:
Where have you got the dates from? I can't find them on the website

Reply:
https://www.droidcon.com/events/droidcon-san-francisco-2023/

Reply:
Shoudl we update the link to this one for the time being?